src/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -8,6 +8,12 @@
     <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release</Configurations>
     <TestRuntime>true</TestRuntime>
   </PropertyGroup>
+  <ItemGroup>
+    <TargetingPackExclusions Include="System.Diagnostics.Debug" />
+    <ReferenceFromRuntime Include="System.Private.CoreLib" />
+    <ReferenceFromRuntime Include="System.Runtime" />
+    <ReferenceFromRuntime Include="System.Threading" />
+  </ItemGroup>
```

Comment:
Adding this so Diagnostics test project can have access to Debug.SetProvider API without it being in ref project contract (since it's not yet API approved)